rpms/java.pan

```diff
@@ -0,0 +1,9 @@
+unique template rpms/java;
+
+variable JAVA_VERSIONS ?= list('11');
+
+'/software/packages' = {
+    foreach (i; version; JAVA_VERSIONS) {
+        pkg_repl(format('java-%s-openjdk', version));
+    };
```

Comment:
What happens to `'/software/packages'` if the `JAVA_VERSIONS` list is somehow empty?

Reply:
You are right, I'll add an explicit exit value to ensure that doesn't return `undef` if it was previously defined.